ister.py

```diff
@@ -447,6 +455,21 @@ def copy_os(args, template, target_dir):
     if args.fast_install:
         run_command("rm -rf {0}".format(args.statedir))
 
+
+def copy_os_dnf(args, template, target_dir):
```

Comment:
Needs unit test

Reply:
I just added tests for the dnf command generation.